### DISCUSSION TITLE
1:1 zoom button

### DISCUSSION THREAD

**Discussion by patrickBR90** [woelper/oculante#142] [Ideas]
a 1:1 zoom should be added, as most images viewers have as a option/button in them.
this could also be a hotkey
example: win photo viewer  
          
![Captura de tela 2023-02-19 003758](https://user-images.githubusercontent.com/89426040/219914816-6809f6bf-9d36-485d-bca0-a17d86c0cb38.png)
![Captura de tela 2023-02-19 003948](https://user-images.githubusercontent.com/89426040/219915143-cb716ae9-74c0-4abd-a754-86675847a92e.png)

**Comment by woelper** [comment]
Great suggestion. While at it, maybe I should add more shortcuts for 200% etc.

**Reply by patrickBR90** [reply]
use <,> or [ ,] for 100% and 200% zoom

**Reply by woelper** [reply]
I now went with the number keys (1,2,3,4,5) but you can easily remap them in settings. It was a bit more complicated since I wanted to keep the current image position under the cursor. Also I will use the opportunity to make this a button and add it under a new submenu to reduce clutter.
